time_tracker_api/time_entries/time_entries_dao.py

```diff
@@ -0,0 +1,311 @@
+import abc
+from commons.data_access_layer.cosmos_db import (
+    CosmosDBDao,
+    CustomError,
+)
+from utils.extend_model import (
+    add_project_info_to_time_entries,
+    add_activity_name_to_time_entries,
+    create_custom_query_from_str,
+    add_user_email_to_time_entries,
```

Comment:
This function is not used so you can remove it
```
add_user_email_to_time_entries
```

Reply:
Done